### PR TITLE
[SMALLFIX] Track source for alluxio-site.properties loaded from classpath

### PR DESCRIPTION
**Pull request by aaudiber**
Otherwise the site properties source will show up as `null` when it's loaded from the classpath

### CODE REVIEW

File: core/common/src/main/java/alluxio/util/ConfigurationUtils.java [hunk 1]
```diff
@@ -41,25 +42,27 @@
...
-   * @param resourceName filename of the properties file
+   * @param resource url of the properties file
    * @return a set of properties on success, or null if failed
    */
-  public static Properties loadPropertiesFromResource(String resourceName) {
-    Properties properties = new Properties();
-
-    InputStream inputStream =
-        Configuration.class.getClassLoader().getResourceAsStream(resourceName);
+  public static Properties loadPropertiesFromResource(URL resource) {
```
Comment (apc999) [inline core/common/src/main/java/alluxio/util/ConfigurationUtils.java]: `@Nullable`

File: core/common/src/main/java/alluxio/util/ConfigurationUtils.java [hunk 2]
```diff
@@ -41,25 +42,27 @@
...
     if (inputStream == null) {
       return null;
     }
 
+    Properties properties = new Properties();
     try {
       properties.load(inputStream);
     } catch (IOException e) {
-      LOG.warn("Unable to load default Alluxio properties file {} : {}", resourceName,
+      LOG.warn("Unable to load default Alluxio properties file {} : {}", resource,
```
Comment (apc999) [inline core/common/src/main/java/alluxio/util/ConfigurationUtils.java]: this is no more default Alluxio properties file. update the warn message

File: core/common/src/main/java/alluxio/util/ConfigurationUtils.java (hunk 1 above)
Comment (apc999) [reply core/common/src/main/java/alluxio/util/ConfigurationUtils.java]: possible to consolidate this one with `loadPropertiesFromFile` so we maintain less code?
e.g., consolidate into one method takes an input stream ?

File: core/common/src/main/java/alluxio/util/ConfigurationUtils.java (hunk 2 above)
Comment (aaudiber) [reply core/common/src/main/java/alluxio/util/ConfigurationUtils.java]: updated, thanks

File: core/common/src/main/java/alluxio/util/ConfigurationUtils.java (hunk 1 above)
Comment (aaudiber) [reply core/common/src/main/java/alluxio/util/ConfigurationUtils.java]: refactored and added `@Nullable`

File: core/common/src/main/java/alluxio/util/ConfigurationUtils.java [hunk 3]
```diff
@@ -41,48 +42,50 @@
...
-        Configuration.class.getClassLoader().getResourceAsStream(resourceName);
-    if (inputStream == null) {
-      return null;
-    }
-
+  @Nullable
+  public static Properties loadPropertiesFromResource(URL resource) {
     try {
-      properties.load(inputStream);
+      return loadProperties(resource.openStream());
```
Comment (apc999) [inline core/common/src/main/java/alluxio/util/ConfigurationUtils.java]: @aaudiber  post-merge comment: don't we need to close this stream? Perhaps a try-with-resource block will help here?

**Pull request merged**